content/en/continuous_delivery/deployments/argocd.md

```diff
@@ -146,7 +146,7 @@ metadata:
 
 ## Visualize deployments in Datadog
 
-The [**Deployments**][6] and [**Deployment Executions**][7] pages populate with data after a deployment is executed. For more information, see [Search and Manage][9] and [CD Visibility Explorer][10].
+The [**Deployments**][6] and [**Executions**][7] pages populate with data after a deployment is executed. For more information, see [Search and Manage][9] and [CD Visibility Explorer][10].
```

Comment:
It looks like this page is still called **Deployment Executions** in the UI. Is there a reason for dropping `Deployment` from the description here?

Reply:
This was a hard one (and explains my logic for the following questions), but I followed what the dropdown menu in the navigation said, not what the tab names are. Happy to revert it if you feel this change is confusing, though I'm not sure how this will end up in the future.

![Screenshot 2024-04-26 at 8 21 57 AM](https://github.com/DataDog/documentation/assets/76412946/df040e6c-df6e-4521-8448-1a7825991aff)